crossbar/common/checkconfig.py

```diff
@@ -1298,7 +1298,8 @@ def check_web_path_service_static(personality, config):
         check_dict_args({
             'enable_directory_listing': (False, [bool]),
             'mime_types': (False, [Mapping]),
-            'cache_timeout': (False, list(six.integer_types) + [type(None)])
+            'cache_timeout': (False, list(six.integer_types) + [type(None)]),
+            'enable_fallback': (False, [str])
         }, config['options'], "'options' in Web transport 'static' path service")
```

Comment:
minor nit-pick, but it's nice to have trailing commas on stuff (so e.g. NEXT time this is changed, there's only a one-line diff ;) )

Reply:
No, good point, my natural reaction is that a comma is superfluous. however it's definitely much neater in PR's just to see one line rather than two ... fixed :)